packages/runtime/src/traits/core/event.tsx

```diff
@@ -55,6 +55,11 @@ const useEventTrait: TraitImplementation<{
           name: 'click',
         });
       },
+      onLoad() {
+        eventHandler({
+          name: 'onLoad',
+        });
+      },
```

Comment:
The lastest main branch has supported cutsom dynamic event names. So now we don't need hardcode here.

Reply:
got it